proxy/src/serverless/conn_pool_lib.rs

```diff
@@ -153,9 +233,11 @@ impl<C: ClientInnerExt> EndpointConnPool<C> {
 
         // do logging outside of the mutex
         if returned {
-            info!(%conn_id, "pool: returning connection '{conn_info}' back to the pool, total_conns={total_conns}, for this (db, user)={per_db_size}");
+            info!(%conn_id, "{}: returning connection '{conn_info}' back to the pool, total_conns={total_conns},
+                                 for this (db, user)={per_db_size}", pool_name);
         } else {
-            info!(%conn_id, "pool: throwing away connection '{conn_info}' because pool is full, total_conns={total_conns}");
+            info!(%conn_id, "{}: throwing away connection '{conn_info}' because pool is full,
+                                 total_conns={total_conns}", pool_name);
```

Comment:
```suggestion
            info!(%conn_id, "{pool_name}: throwing away connection '{conn_info}' because pool is full, total_conns={total_conns}");
```